@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { VectorTileDataSource } from "@here/harp-vectortile-datasource";
+
+/**
+ * `OmvDataSource` is used for the visualization of vector tiles
+ * provided in the OMV format.
+ *
+ * @example
+ * ```typescript
+ *    const dataSource = new OmvDataSource({
+ *        baseUrl: "https://vector.hereapi.com/v2/vectortiles/base/mc",
+ *        authenticationCode: apikey
+ *    });
+ *    mapView.addDataSource(dataSource);
+ *   ```
+ */
+export class OmvDataSource extends VectorTileDataSource {}
```

Comment:
Can you mark it as deprecated or do we keep it as a utility class for easier HERE Vector Tile initialization?

Reply:
Good idea, I will mark all the symbols as deprecate in a follow up change.